src/main/java/com/baidu/hugegraph/backend/store/tikv/TikvStdSessions.java

```diff
@@ -254,9 +280,73 @@ public void rollback() {
         }
 
         @Override
-        public Pair<byte[], byte[]> keyRange(String table) {
-            // TODO: get first key and lastkey of fake tikv table
-            return null;
+        public List<Pair<byte[], byte[]>> keyRanges(String table) {
+            E.checkArgument(StringUtils.isNotBlank(table),
+                            "Table don't allow empty");
+
+            ByteString startKey = this.key(table);
+            ByteString endKey = ByteString.copyFrom(
+                                Key.toRawKey(startKey).nextPrefix().getBytes());
+            List<TiRegion> regions = this.fetchRegionsFromRange(startKey,
+                                                                endKey);
```

Comment:
may fetchRegionsFromRange return a region which is out of the endKey?

Reply:
if return an out of the endkey, the query will return other table data when use the out of endkey params